cmd/csaf_checker/main.go

```diff
@@ -132,10 +135,14 @@ func main() {
 	domains, err := flags.Parse(opts)
 	errCheck(err)
 
-	if len(domains) == 0 {
+	if len(domains) == 0 && !opts.Version {
```

Comment:
From the sidelines: To me that smells a bit. Why not change the flow by first doing the version request dance and then one can spare relating surprisingly the domains and the request for version data dimensions.